### ISSUE TITLE
[Request] Discover Movies / Discover TV

### ISSUE THREAD

**Issue by dannyhanes** [lokenx/plexrequests-meteor#226]
It would be awesome if there were a discovery tab of some sort where I could browse new movie releases, top rated movies, top rated tv shows for the year etc. and add them just like I would any other movie / tv show.

Maybe scrape from: 
http://www.rottentomatoes.com/top-tv/
http://www.rottentomatoes.com/top/

**Comment by lokenx** [comment]
Cool idea, a bit like what Couch Potato offers?

**Comment by dannyhanes** [comment]
Yeah, exactly. The way I have stuff set up, I am running couchpotato and sickbeard, but my goal is to only navigate to plexrequests. This makes it wife-friendly haha. 

I use couchpotato and sickbeard for managing downloads and configuring settings, but requests for just adding media.

**Comment by DejaVu** [comment]
+1 for this! Such a great idea and keeps everyone in the same place on PlexRequests!

**Comment by overdox** [comment]
+1 for this.

**Comment by SoulFear** [comment]
Yes please! :+1:

**Comment by rigrassm** [comment]
Movie party of this request was added with #461.
